packages/react-dom/src/shared/assertValidProps.js

```diff
@@ -31,7 +33,7 @@ function assertValidProps(tag: string, props: ?Object) {
       '%s is a void element tag and must neither have `children` nor ' +
         'use `dangerouslySetInnerHTML`.%s',
       tag,
-      __DEV__ ? ReactDebugCurrentFrame.getStackAddendum() : '',
+      __DEV__ && ReactDebugCurrentFrame ? ReactDebugCurrentFrame.getStackAddendum() : '',
```

Comment:
We don't expect it to ever be null in dev. Can assert somehow?

Reply:
Can we maybe reduce it to just `ReactDebugCurrentFrame ? ReactDebugCurrentFrame.getStackAddendum() : '',` since `ReactDebugCurrentFrame` only exists in dev anyway?